File: kickstart.py
def speed_typing(originial,real):
    if(len(orginal)==len(real)):
        if(orginial==result):
            return 0
        else:
            return -1
    elif(len(orginal)<len(real)):
        if(orginial in result):
            return len(real)-len(orginal)
        else:
            return -1;
    else:
        return -1;


def charcheck(orginal,real):
    count=0
    if(len(orginal)<len(real)):
        for i in orginal:
            if(i not in real):
                count+=1
    print(count)


char

t=int(input())
l=[]
for i in range(t):
    original = input()
    real=input()
    l.append(speed_typing(originial,real))
print(l)

File: Test FIles/kickstart.cpp
#include <iostream>
using namespace std;

int speed_typing(string orginial, string result)
{

    if (orginial.length() == result.length())
    {
        if (orginial == result)
        {
            return 1;
        }
        else{
            return -1;
        }
    }
    else if (orginial.length()<=result.length())
    {
        int count;
        for (int i=0; i<orginial.length(); i++)
        {
            if(orginial[i]!=result[i])
            {

            }
        }
    }
    
}

int main()
{
    int t;
    cin >> t;
    int arr[t];
    for (int i = 0; i < t; i++)
    {
        string orginial, result;
        arr[i] = speed_typing(orginial, result);
    }
}